### ISSUE TITLE
Add coverity github workflow?

### ISSUE THREAD

**Issue by gdevenyi** [zlib-ng/zlib-ng#807]
https://scan.coverity.com/github

**Comment by gdevenyi** [comment]
#805 at least was found by it

**Comment by nmoinvaz** [comment]
It is already configured, but has limited access.

**Comment by Dead2** [comment]
We already have an active coverity profile with zlib-ng, but last time we tried to add it to CI, it ended in a mess. So now I just run manual checks when I feel we have changes that need checking or before releases.

I also don't think Coverity has any integration into Github Actions, only Travis CI, but I may be wrong. When we tried to add it, we used Travis, and there are several issues that need consideration with using it in CI. Both security and limitations on number of scans, so when we had it, I had to do a manual push into a specific branch.
Quickly found out it was better to just run it manually locally to get better control over the process.
